forge-gui/res/cardsfolder/s/space_marine_scout.txt

```diff
@@ -4,8 +4,8 @@ Types:Creature Astartes Scout
 PT:2/1
 K:First Strike
 K:Vigilance
-T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | CheckSVar$ X | SVarCompare$ GTY | ValidCard$ Card.Self | Execute$ TrigChange | TriggerDescription$ Concealed Position — When CARDNAME enters the battlefield, if an opponent controls more lands than you, you may search your library for a Plains card, put it onto the battlefield tapped, then shuffle.
-SVar:TrigChange:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | ChangeType$ Land.Plains | ChangeNum$ 1 | Tapped$ True
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | CheckSVar$ X | SVarCompare$ GTY | ValidCard$ Card.Self | OptionalDecider$ You | Execute$ TrigChange | TriggerDescription$ Concealed Position — When CARDNAME enters the battlefield, if an opponent controls more lands than you, you may search your library for a Plains card, put it onto the battlefield tapped, then shuffle.
+SVar:TrigChange:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | ChangeType$ Land.Plains | ChangeNum$ 1 | Tapped$ True | ShuffleNonMandatory$ True
```

Comment:
what makes you think it should have ShuffleNonMandatory?

Reply:
nevermind :)

Reply:
The ability on the Scout is closest to the relevant ones on [Land Tax](https://scryfall.com/card/cmm/37/land-tax) or [Knight of the White Orchid](https://scryfall.com/card/moc/193/knight-of-the-white-orchid), chiefly by being a "may" ability. Both of those card scripts have `ShuffleNonMandatory` and since this one also missed the `OptionalDecider$` I figured this was initially lifted from [Loyal Warhound](https://scryfall.com/card/afr/23/loyal-warhound) or a similar card where this type of enters effect is mandatory. Then again, I'm only going off existing scripts and not fully aware of what's to be deprecated moving forward.

Reply:
let's talk on Discord sometime if we have time between sets for what's clean up worthy etc.